[WEBSITE-716] - Update the Jenkins officer role descriptions (#2943)
jenkins-infra/jenkins.io: content/project/team-leads.adoc

---
layout: simplepage
title: "Jenkins Team Leads and Officers"
section: project
---

> Jenkins team leads and officers are individual contributors have the responsibility and accountability for

Reviewer: ```suggestion
Jenkins team leads and officers are individual contributors with the responsibility and accountability for
```

leading a particular facet of the Jenkins project and/or the community.
These roles are held by community members who have proven themselves to be responsible and courteous,
representing the executive branch of the Jenkins project governance model.

> Team leads and officers act with the authority in guiding and making decisions within the project, and representing the Jenkins project within the area of their responibility.

Reviewer: ```suggestion
Team leads and officers act with the authority in guiding and making decisions within the project, and representing the Jenkins project within the area of their responsibility.
```

Officer roles are an essential part of Jenkins' community governance and well-being.

> The difference between _Jenkins Officers_ and _Team Leads_ lays in the appointement process:

Reviewer: ```suggestion
The difference between _Jenkins Officers_ and _Team Leads_ lays in the appointment process:
```

  Jenkins Officers have a fixed term though the number of terms is not limited.

## Responsibilities

All Jenkins officers and team leads have the following responsibilities:

* Coordinating the work inside the teams and SIGs they lead.
  While team leads are responsible to ensure that the mission-critical community processes work, they are not expected to work on them single-handedly.
* Onboarding new contributors to the teams they lead.
* Mentorship and knowledge sharing within the team to ensure healthy officer rotation in the project,
  with opportunity for team leads to move on to other roles or to take a break when needed.
* Facilitating initiatives within the scope of the responsibility, maintaining the team's roadmap,
  and ensuring that key team initiatives reflected on the link:/project/roadmap/[public Jenkins roadmap].
* Representing the Jenkins project and community within the area of their responsibility.

## Team Leads and Officers

[[security]]
### Security Officer

Coordinates the security-related activities in the Jenkins project. This is an **elected role**. 

* Run the Jenkins CERT meeting

> * link:https://wiki.jenkins.io/display/JENKINS/Rewards+for+reporting+security+issues[Manage sending gifts to qualifying reporters of resolved security issues]

Reviewer: ```suggestion
* link:https://www.jenkins.io/security/gift/[Manage sending gifts to qualifying reporters of resolved security issues]
```

Reviewer: Also line 46 should link to https://www.jenkins.io/security/advisories/

Author: Probably should be removed at all. 
I would rather refer "Maintain the vulnerability reporting process" or so on.

Reviewer: Probably.

Author: ```suggestion
* link:/security/gift/[Manage sending gifts to qualifying reporters of resolved security issues]
```

Will copy-edit later

* Represent the Jenkins project on security topics with third parties
* Represent the https://www.jenkins.io/security/cna/[Jenkins CNA].

[[infrastructure]]
### Infrastructure Officer

Oversees the Jenkins project infrastructure. This is an **elected role**.

* Direct triage of INFRA issues
* Direct maintenance of Jenkins project infrastructure ('INFRA' issues)
* Encourage contributors to INFRA project
* Communicate maintenance windows and manage execution of those windows
* Coordinate Jenkins Infrastructure initiatives and define priorities, get key initiatives reflected on the link:/project/roadmap/[public Jenkins roadmap]

[[events]]
### Events Officer

_note: This team lead position was approved in the link:http://meetings.jenkins-ci.org/jenkins-meeting/2016/jenkins-meeting.2016-02-17-19.00.html[2015-02-17 governance meeting]_

Has an active role in coordinating Jenkins community events and outreach programs.
Such events include conferences where Jenkins is represented (cdCon, FOSDEM, SCALE, etc.),
community-organized local and virtual meetups,
contributor summits,
outreach events (hackathons, Hacktoberfest, GSoC/GSoD, etc.), and speaking opportunities.

> This is an **elected role**. 

Reviewer 2: For the Infrastructure Officer, the information that it is an elected role is right under the title and next to a simple description. I think it would be good to maintain that for all officers so that people don't have to search to determine if the role is elected or appointed.

> Has an active role in coordinating Jenkins community events and outreach programs.
> Such events include conferences where Jenkins is represented (cdCon, FOSDEM, SCALE, etc.),
> community-organized local and virtual meetups,
> contributor summits,
> outreach events (hackathons, Hacktoberfest, GSoC/GSoD, etc.), and speaking opportunities.
> This is an **elected role**. 

Author: ```suggestion
Has an active role in coordinating Jenkins community events and outreach programs. This is an **elected role**. 

Jenkins events include conferences where Jenkins is represented (cdCon, FOSDEM, SCALE, etc.),
community-organized local and virtual meetups,
contributor summits,
outreach events (hackathons, Hacktoberfest, GSoC/GSoD, etc.), and speaking opportunities.
```

* Drive security policy definition/changes in the community

> * Represent the Jenkins project on security topics with third parties

Reviewer: ```suggestion
* Represent the Jenkins project on security topics with third parties
* Represent the https://www.jenkins.io/security/cna/[Jenkins CNA].
```
